### DISCUSSION TITLE
"Used By" section is not populated

### DISCUSSION THREAD

**Discussion by edoardottt** [community/community#74579] [Repositories] [answered]
### Select Topic Area

Bug

### Body

As you can see here -> https://github.com/edoardottt/depsdev/network/dependents there are some dependents on this repo. However in the repo main page this is not shown -> https://github.com/edoardottt/depsdev.

References: 
- https://github.com/orgs/community/discussions/34687

**Comment by nuxy** [comment]
I have been experiencing this issue **since forever**. :sob:

The only workaround I found was by loading the [Dependency Graph / Dependents](https://github.com/nuxy/slot-machine-gen/network/dependents) page would sometimes result with the ["Used by" section](https://github.com/nuxy/slot-machine-gen) on the project profile being populated.  However, as of recently this workaround no longer works and has left me dumbfounded how long this issue has existed without being addressed.

**Comment by nuxy** [comment, marked as answer]
Contacted support on 11/23 regarding this issue.  Their reply is below.

> Thank you for contacting GitHub support. GitHub recently changed it so the Used by section will only show on repositories where the selected package has at least 100 dependents, as documented [here](https://docs.github.com/en/code-security/supply-chain-security/understanding-your-software-supply-chain/exploring-the-dependencies-of-a-repository#changing-the-used-by-package).

### References
- https://docs.github.com/en/code-security/supply-chain-security/understanding-your-software-supply-chain/exploring-the-dependencies-of-a-repository#changing-the-used-by-package
